tests/ext/test_dictpersistence.py

```diff
@@ -25,33 +25,33 @@
 
 
 @pytest.fixture(autouse=True)
-def reset_callback_data_cache(cdc_bot):
+def _reset_callback_data_cache(cdc_bot):
     yield
     cdc_bot.callback_data_cache.clear_callback_data()
     cdc_bot.callback_data_cache.clear_callback_queries()
 
 
-@pytest.fixture(scope="function")
+@pytest.fixture()
 def bot_data():
     return {"test1": "test2", "test3": {"test4": "test5"}}
 
 
-@pytest.fixture(scope="function")
+@pytest.fixture()
 def chat_data():
     return {-12345: {"test1": "test2", "test3": {"test4": "test5"}}, -67890: {3: "test4"}}
 
 
-@pytest.fixture(scope="function")
+@pytest.fixture()
 def user_data():
     return {12345: {"test1": "test2", "test3": {"test4": "test5"}}, 67890: {3: "test4"}}
 
 
-@pytest.fixture(scope="function")
+@pytest.fixture()
 def callback_data():
     return [("test1", 1000, {"button1": "test0", "button2": "test1"})], {"test1": "test2"}
 
 
-@pytest.fixture(scope="function")
```

Comment:
why all these changes?

Reply:
the default scope is actually function, so I think that tool just prefers it to omit defining it explicitly 🤷🏽